installer/mac/SymphonySettingsPlugin/SymphonySettingsPlugin/MyInstallerPane.m

```diff
@@ -31,7 +31,7 @@ - (BOOL)shouldExitPane:(InstallerSectionDirection)dir {
     }
     
     // Now, validate the url against a url regex
-    NSString *regex = @"^((?:http:\/\/)|(?:https:\/\/))(www.)?((?:[a-zA-Z0-9]+\.[a-z]{3})|(?:\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(?::\d+)?))([\/a-zA-Z0-9\.]*)$";
+    NSString *regex = @"^((?:http:\/\/)|(?:https:\/\/))(www.)?((?:[a-zA-Z0-9\-]+\.[a-z]{3})|(?:\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(?::\d+)?))([\/a-zA-Z0-9\-.]*)$";
```

Comment:
@KiranNiranjan You may have to change the regex to support all characters listed in the RFC spec. Please take a look here:

http://www.ietf.org/rfc/rfc3986.txt

This may help -> https://stackoverflow.com/questions/1856785/characters-allowed-in-a-url